frugen.c

```diff
@@ -264,6 +319,9 @@ int main(int argc, char *argv[])
 		/* Set input file format to JSON */
 		{ .name = "json",          .val = 'j', .has_arg = false },
 
+		/* Set input file format to binary */
+		{ .name = "binaryformat",          .val = 'x', .has_arg = false },
```

Comment:
please make it just `"binary"`

Reply:
sorry, forgot that `--binary` is already used. Let it be `--raw` then, with `-r` for short.